MovieApp/MovieApp/MoviesViewController.m

```diff
@@ -19,8 +26,17 @@ @implementation MoviesViewController
 - (void)viewDidLoad {
     
     [super viewDidLoad];
+    [self configureView];
+
+    downloader = [[MovieDBDownloader alloc] init];
+    [downloader configure];
+    [self.sortSegmentedControl setSelectedSegmentIndex:2];
+    [self.sortSegmentedControl sendActionsForControlEvents:UIControlEventValueChanged];
```

Comment:
Add left and right padding to segmented control also (same as for collection view). You can leave it like this, since it's native design.